pkg/inference/preset-inferences.go

```diff
@@ -138,7 +139,25 @@ func CreatePresetInference(ctx context.Context, workspaceObj *kaitov1alpha1.Work
 		volumeMounts = append(volumeMounts, adapterVolumeMount)
 	}
 
-	commands, resourceReq := prepareInferenceParameters(ctx, inferenceObj)
+	skuHandler, err := utils.GetSKUHandler()
+	if err != nil {
```

Comment:
Can you use a common function for GetSKUGPU? Instead of copying the same code in two places?